firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -370,4 +383,28 @@ private boolean isMultiValueFilter(Target target, FieldPath fieldPath) {
   private byte[] encodeFieldIndex(FieldIndex fieldIndex) {
     return serializer.encodeFieldIndex(fieldIndex).toByteArray();
   }
+
+  // TODO(indexing): Add support for fetching last N updated entries.
+  public List<FieldIndex> getFieldIndexes() {
+    List<FieldIndex> allIndexes = new ArrayList<>();
+    db.query(
+            "SELECT index_id, collection_group, index_proto, update_time_seconds, update_time_nanos FROM index_configuration "
+                + "WHERE active")
```

Comment:
Does `WHERE active` actually work? And even if it works, we should still be using `active = 1`.

Reply:
Thanks for catching.  SQL query still works without adding `= 1`, but it's best to follow best practices.